### ISSUE TITLE
New Markdown files must start with Heading Level 2 because Book Title is Heading level 1

### ISSUE THREAD

**Issue by zDEFz** [rust-lang/mdBook#2372]
### Problem

There must be only one Heading Level one present. 
If Book Title is shown, its H1. Then all Markdown files need to Start with H2.

### Steps

1. Set book title
2. Make entries in Summary.md
3. Find that the entries all start with H1 "#" instead of H2 "##"

### Possible Solution(s)

Start new Markdown files with H2 "##"

### Version

```text
mdbook v0.4.37
```

**Comment by chriskrycho** [comment]
This is indeed a mild accessibility issue (though not a WCAG AA violation!), but the better solution is probably to make the title of the book on each page a styled link that is *not* a heading; see [this discussion](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/#is-the-logo-a-heading%3F) (among many others!) for example.

**Comment by zDEFz** [comment]
> This is indeed a mild accessibility issue (though not a WCAG AA violation!), but the better solution is probably to make the title of the book on each page a styled link that is _not_ a heading; see [this discussion](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/#is-the-logo-a-heading%3F) (among many others!) for example.

You still need to expose language of the page, and the H1 must always reflect the site title - for which can only 1 h1 exist.
If you decide to not make the Book Title H1, then the site title is also missing, because currently H1 is automagically the site title. But you need to have one.

**Comment by chriskrycho** [comment]
A `<title>` works just fine for that, and the `h1` should, as a rule, *not* be the same across all pages on a site!

**Comment by zDEFz** [comment]
> A `<title>` works just fine for that, and the `h1` should, as a rule, _not_ be the same across all pages on a site!

That is a bit imprecise. Refer to https://dequeuniversity.com/rules/axe/4.0/document-title

**Comment by chriskrycho** [comment]
Yep! In this case the `title` should probably be something like `<Chapter>: <Section> – <Book>`. My point was not that the title would be *only* the book title but that it would include and so subsume it.
